ngrinder-core/src/main/java/org/ngrinder/monitor/share/domain/SystemInfo.java

```diff
@@ -60,15 +60,11 @@ public boolean isParsed() {
 
 	protected BandWidth bandWidth;
 
-	private long totalCpuValue;
-
-	private long idleCpuValue;
-
 	private long freeMemory;
 
 	private long totalMemory;
 
-	private float cpuUsedPercentage;
+	private double cpuUsedPercentage;
```

Comment:
Is it compatible when map into old agents?

Reply:
Since 3.5.0 old agent is automatically updated.

Reply:
Fixed